### PR TITLE
pash: init at git-2015-11-06

### PR DESCRIPTION
**Pull request by ForNeVeR**
I've made the package for open source PowerShell-like Linux shell. Applications developers are notified (and I am one of them), the package has been tested.

### CODE REVIEW

File: pkgs/shells/pash/default.nix [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  version = "git-2015-11-06";
+  
+  src = fetchFromGitHub {
+    owner = "Pash-Project";
+    repo = "Pash";
+    rev = "50695a28eaf6c8cbfdc8ecddd91923c64e07b618";
+    sha256 = "17hs1f6ayk9qyyh1xsydk46n6na7flh2kbd36dynk86bnda5d3bn";
+  };
+
+  preConfigure = "rm -rvf Source/PashConsole/bin/*";
```
Comment (jagajaga) [inline pkgs/shells/pash/default.nix]: Use `$src`

Comment (ForNeVeR) [inline pkgs/shells/pash/default.nix]: Thanks for the review, I've used it.

**Comment by cillianderoiste** [comment]
Looks good, thanks!

**Comment by ForNeVeR** [comment]
And thank you, guys.

**Pull request merged**